README.md

```diff
@@ -1,8 +1,65 @@
-# github-applets
-Extending GitHub Enterprise with Serverless Functions
+# What problem are we trying to solve?
+We need a solution that allows our SI partners to automate SDLC workflows based on industry vertical needs.  These workflows and toolchains are currently being called _blueprints_ inside our partner community.  
 
-# WIP
-This repo has the goal to host some initial POC scripts to illustrate a few reference use cases for Applets.
+Currently, integrating GitHub Enterprise into the SDLC requires the assumption that **every** SDLC workflow begins on GitHub Enterprise.  Our business partners see GitHub Enterprise as one of many products inside their customer's overall toolchain purchase.  As GitHub Enterprise evolves into a **Platform**, we need a way to simplify building integrations and workflows that can connect the overall SDLC.
```

Comment:
>we need a way to simplify building integrations and workflows that can connect the overall SDLC.

I think the notion here is that we kickstart the idea of building an integrated SDLC toolchain?